dev/com.ibm.ws.cdi.1.2.ejb/bnd.bnd

```diff
@@ -21,9 +21,7 @@ Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
   com.ibm.ws.cdi.ejb.security.EjbSecurityContextStore,\
   com.ibm.ws.cdi.ejb.interceptor.liberty.JCDIHelperFactoryImpl
 
-
-Service-Component: \
- com.ibm.ws.cdi.1.2.ejb.deserializationClassProvider;\
+Service-Component: com.ibm.ws.cdi_1_2.ejb.deserializationClassProvider;\
```

Comment:
Why the change to underscores?

Reply:
The component fully qualified name validator now considers numbers after a period invalid. Would you like to use a different format? It could be `com.ibm.ws.cdi12.ejb.deserializationClassProvider`. For names that have bVersion, they currently look like `${Bundle-SymbolicName}_1_0_24.*`.

Reply:
no, just curious